api-report/driver-base.api.md

```diff
@@ -82,7 +83,7 @@ export class DocumentDeltaConnection extends EventEmitterWithErrorHandling<IDocu
     // (undocumented)
     protected readonly socket: Socket;
     submit(messages: IDocumentMessage[]): void;
-    submitSignal(message: IDocumentMessage): void;
+    submitSignal(content: unknown, targetClientId?: string): void;
```

Comment:
again, why change to unknown, was IDocumentMessage wrong to being with?

Reply:
Yes, it's never been a `IDocumentMessage`. The arg is the `contents` property of `ISignalMessage`. It should have been `unknown` or `any`

Reply:
Chatted offline. I will undo the typing changes in this PR and follow up with another PR in `next` to correct the signal typings since those will be breaking changes.